components/backup/src/endpoint.rs

```diff
@@ -580,6 +700,21 @@ fn backup_file_name(store_id: u64, region: &Region, key: Option<String>) -> Stri
         ),
     }
 }
+pub fn rawkv_cf(cfname: &str) -> Result<CfName> {
+    if cfname.is_empty() {
+        return Ok(CF_DEFAULT);
+    }
+    for c in DATA_CFS {
+        if cfname == *c {
+            return Ok(c);
+        }
+    }
+    let mut info = String::from("invalid cf name ");
+    info.push_str(cfname);
+    Err(errors::Error::Storage(<StorageError>::from(
+        StorageErrorInner::InvalidCf(info),
+    )))
+}
```

Comment:
Consider use the one in storage module instead of duplicating it?

Reply:
> @xinhua5 please fix the error below,
> 
> ```
> let stat = match brange.backup(&mut writer, &engine, backup_ts) {
> 
>  ^^^^^^ expected 4 parameters
> ```

yes

Reply:
use current code for the time being